### PR TITLE
overhaul of the readme / .org listing

### PR DESCRIPTION
**Pull request by zbtirrell**
This is a complete overhaul of the language in our readme to improve how
it appears on our wordpress.org listing page.

### CODE REVIEW

**Comment by zbtirrell** [comment]
I may have a follow-up with some new screenshots too

File: readme.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
 === The Events Calendar ===
 
-Contributors: ModernTribe, borkweb, barry.hughes, bordoni, brianjessee, aguseo, cliffpaulick, courane01, faction23, GeoffBel, geoffgraham, ggwicz, jbrinley, jentheo, leahkoerper, lucatume, mastromktg, MZAWeb, neillmcshea, nicosantos, patriciahillebrandt, peterchester, reid.peifer, roblagatta, ryancurban, shane.pearlman, tribecari, vicskf, zbtirrell, juanfra
+Contributors: ModernTribe, borkweb, barry.hughes, bordoni, brianjessee, aguseo, cliffpaulick, courane01, faction23, GeoffBel, geoffgraham, jbrinley, jentheo, leahkoerper, lucatume, neillmcshea, nicosantos, patriciahillebrandt, peterchester, reid.peifer, roblagatta, ryancurban, shane.pearlman, vicskf, zbtirrell, juanfra
```
Comment (Camwyn) [inline readme.txt:3]:
```suggestion
Contributors: ModernTribe, borkweb, barry.hughes, bordoni, brianjessee, aguseo, camwyn, cliffpaulick, courane01, faction23, GeoffBel, geoffgraham, jbrinley, jentheo, leahkoerper, lucatume, neillmcshea, nicosantos, patriciahillebrandt, peterchester, paulskim, reid.peifer, roblagatta, ryancurban, shane.pearlman, vicskf, zbtirrell, juanfra
```

Comment (Camwyn) [reply readme.txt:3]: Adding some folks who haven't been added yet 😉

**Comment by Camwyn** [comment]
@zbtirrell I created a PR against this branch for some spelling errors that have been hanging around a while: https://github.com/moderntribe/the-events-calendar/pull/3191

**Pull request merged**